com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Accumulation/SubFrameManager.cs

```diff
@@ -106,14 +146,16 @@ internal void EndRecording()
         }
 
         // Should be called before rendering a new frame in a sequence (when accumulation is desired)
-        internal void PrepareNewSubFrame()
+        internal void PrepareNewSubFrame(int camID)
```

Comment:
I think it's better to avoid the camera parameter here (and in the public API) because:
- we don't always have a camera when this is called (for example in the recorder when capturing the game view)
- the rest of the API (BeginRecording, EndRecording) affects all cameras, so it makes more sense to have this call affect all cameras too.

So my suggestion is 
- Put the if statement with the reset() in a for loop (for all cameras)
- For the rest of the function that manipulates the Time.*, find the "max iteration number" and execute the statements for this number. This ensures that we get the correct/expected result, even if we have "stale" cameras that did not render at all.

Reply:
Done: I did take the max of all iterations, although I'd guess we're not supposed to have multiple cameras here: if so, it's not gonna crash, but might mess up the Time settings.

Reply:
It should work correctly even if we have multiple cameras (for example two cameras that render side-by-side or a security camera that appears in the main camera), but the user-side should call PrepareNewSubFrame() only once per frame, so all cameras will get the same weights/motion blur params.